controllers/numaresourcesoperator_controller.go

```diff
@@ -227,7 +227,7 @@ func (r *NUMAResourcesOperatorReconciler) reconcileResource(ctx context.Context,
 	}
 	r.Recorder.Eventf(instance, corev1.EventTypeNormal, "SuccessfulRTECreate", "Created Resource-Topology-Exporter DaemonSets")
 
-	instance.Status.MachineConfigPools = syncMachineConfigPoolNodeGroupConfigStatuses(instance.Name, instance.Status.MachineConfigPools, trees)
+	instance.Status.MachineConfigPools = syncMachinetConfigPoolNodeGroupConfigStatuses(instance.Name, instance.Status.MachineConfigPools, trees)
```

Comment:
was this modified by mistake?  I don't see a matching function

Reply:
yes, a typo sneaked in. Thanks!